releasenotes/notes/max-containerd-spec-5dec95b909062b4e.yaml

```diff
@@ -0,0 +1,4 @@
+---
+security:
+  - |
+    The Agent now checks containerd containers `Spec` size before parsing it. Any `Spec` exceeding 2MB will not be parsed and a warning will be emitted. This impacts the `container_env_as_tags` feature for environments based on `containerd` outside of Kubernetes.
```

Comment:
I think it also breaks `%%hostname%%` in configs because the hostname is in the Spec (`workloadContainer.Hostname = spec.Hostname`).

Reply:
Yes that's correct!